npm run build warnings

NPM run build warnings:

```
> dark-matter@2.2.0 build
> NODE_ENV=production webpack --mode production


✖ Webpack
  Compiled with some errors in 25.72s



      ./node_modules/scheduler/index.js 197 bytes [built] [code generated]
      ./node_modules/scheduler/cjs/scheduler.production.min.js 6.65 KiB [built] [code generated]
  modules by path ./domain-mapping/ui/ 97.8 KiB (javascript) 529 bytes (css/mini-extract)
    ./domain-mapping/ui/index.js + 45 modules 97.7 KiB [built] [code generated] [1 error]
    ./domain-mapping/ui/App.css 50 bytes [built] [code generated]
    css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[2].use[1]!./node_modules/postcss-loader/dist/cjs.js!./domain-mapping/ui/App.css 529 bytes [built] [code generated]

ERROR in ./domain-mapping/ui/Components/DomainRow.js
Module Error (from ./node_modules/eslint-loader/dist/cjs.js):

/var/www/amasan.co.uk/dark-matter/domain-mapping/ui/Components/DomainRow.js
  30:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  32:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  34:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  35:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  36:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  37:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  38:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  39:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  40:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs
  42:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs

✖ 10 problems (10 errors, 0 warnings)

 @ ./domain-mapping/ui/Components/DomainMapping.js 20:0-36 205:52-61
 @ ./domain-mapping/ui/index.js 3:0-55 10:41-54

domain-mapping (webpack 5.65.0) compiled with 1 error in 25726 ms
```

--- a/.editorconfig
+++ b/.editorconfig
@@ -4,7 +4,7 @@ root = true
 charset = utf-8
 end_of_line = lf
 indent_size = 2
-indent_style = space
+indent_style = tab
 insert_final_newline = true
 trim_trailing_whitespace = true
 

--- a/domain-mapping/ui/API/Enums.js
+++ b/domain-mapping/ui/API/Enums.js
@@ -5,5 +5,5 @@
  */
 export const DOMAIN_TYPES = {
 	MAIN: 1,
-	MEDIA: 2
-}
+	MEDIA: 2,
+};

--- a/domain-mapping/ui/Components/DomainRow.js
+++ b/domain-mapping/ui/Components/DomainRow.js
@@ -4,7 +4,7 @@ import React from 'react';
 import DomainDisplayMedia from './DomainDisplayMedia';
 import DomainDisplayPrimary from './DomainDisplayPrimary';
 import DomainDisplaySecondary from './DomainDisplaySecondary';
-import { DOMAIN_TYPES } from "../API/Enums";
+import { DOMAIN_TYPES } from '../API/Enums';
 
 class DomainRow extends React.Component {
 	/**
@@ -21,26 +21,26 @@ class DomainRow extends React.Component {
 		this.props.update( data );
 	};
 
-  /**
-   * Handle converting domains between secondary to media domains and vice versa.
-   *
-   * @param {Object} event
-   */
-  handleConvert = ( event ) => {
-	event.preventDefault();
+	/**
+	 * Handle converting domains between secondary to media domains and vice versa.
+	 *
+	 * @param {Object} event
+	 */
+	handleConvert = ( event ) => {
+		event.preventDefault();
 
-	const data = { ...this.props.domain };
+		const data = { ...this.props.domain };
 
-	if ( DOMAIN_TYPES.MAIN === data.type ) {
-		/** Convert media domain to secondary domain. */
-		data.type = 2;
-	} else if ( DOMAIN_TYPES.MEDIA === data.type ) {
-		/** Convert media domain to secondary domain. */
-		data.type = 1;
+		if ( DOMAIN_TYPES.MAIN === data.type ) {
+			/** Convert media domain to secondary domain. */
+			data.type = 2;
+		} else if ( DOMAIN_TYPES.MEDIA === data.type ) {
+			/** Convert media domain to secondary domain. */
+			data.type = 1;
+		}
+
+		this.props.update( data );
 	}
-
-	this.props.update( data );
-  }
 
 	/**
 	 * Handle the Deleting of the domain.

